tests/resources/generator/algebraic_eqn_computed_var_on_rhs/code.c

```diff
@@ -15,6 +15,8 @@ struct VariableInfo {
     char units[14];
 };
 
+const struct VariableInfo VOI_INFO = {"", "", ""};
+
```

Comment:
If there is no VOI then I wouldn't expect to see this.

Reply:
We need to know whether a model has a VOI or not. So, I thought I would rely on what we currently generate to generate something that could be used to be determine whether we have a VOI, hence an "empty" `VariableInfo`. The alternative would be to have something like:

```
const size_t HAS_VOI = 1;
```

or

```
const size_t HAS_VOI = 0;
```

but I thought that reusing what we already have would be easier?...

Reply:
I think

```
STATE_COUNT = 0
```
 Gives us that information.

Reply:
Doh! Of course! 🤦

Ok, will (obviously) do that!

Reply:
Otherwise... shouldn't we do something similar for (our new) `STATE_INFO` and `VARIABLE_INFO`? I mean, we should probably not generate `STATE_INFO` if there are no states and `VARIABLE_INFO` if there are no variables...?

Reply:
Sure for the state info, but are there any cases where you can have an equation without a variable?

Reply:
No, obviously. However, I would personally always generate code for `VOI_INFO`, `STATE_INFO` and `VARIABLE_INFO`, even if `VOI_INFO` and `STATE_INFO` are to be "empty". It's just for consistency throughout. In fact, it's like for the `algebraic_eqn_computed_var_on_rhs` test, the `computeRates()` and `computeVariables()` methods are generated even though they are empty.